blivet/static_data/luks_data.py

```diff
@@ -64,6 +66,14 @@ def luks_devs(self):
     def luks_devs(self, value):
         self.__luks_devs = value
 
+    @property
+    def pbkdf_args(self):
+        return self.__pbkdf_args
+
+    @pbkdf_args.setter
+    def pbkdf_args(self, value):
+        self.__pbkdf_args = value
+
```

Comment:
I think a single underscore would suffice for this. I used two for passphrases because of the obfuscation required to access the attributes directly since they are passphrases. Maybe I'm wrong, but my sense is that the PBKDF args don't require such measures.